typescript/agentkit/README.md

```diff
@@ -493,6 +516,36 @@ const client = createWalletClient({
 const walletProvider = new ViemWalletProvider(client);
 ```
 
+#### Configuring ViemWalletProvider gas parameters
+
+The `ViemWalletProvider` also exposes parameters for effecting the gas calculations.
+
+```typescript
+import { ViemWalletProvider } from "@coinbase/agentkit";
+import { privateKeyToAccount } from "viem/accounts";
+import { baseSepolia } from "viem/chains";
+import { http } from "viem/transports";
+import { createWalletClient } from "viem";
+
+const account = privateKeyToAccount(
+  "0x4c0883a69102937d6231471b5dbb6208ffd70c02a813d7f2da1c54f2e3be9f38",
+);
+
+const client = createWalletClient({
+  account,
+  chain: baseSepolia,
+  transport: http(),
+});
+
+const walletProvider = new ViemWalletProvider(client, {
+    gasLimitMultiplier: 2.0,  // Adjusts gas limit estimation
+    feePerGasMultiplier: 2.0, // Adjusts max fee per gas
+});
+```
+
+**Note**: Gas parameters only impact the `walletProvider.sendTransaction` behavior. Actions that do not rely on direct transaction calls, such as `request_faucet_funds`, `deploy_token`, `deploy_contract`, and `native_transfer`, remain unaffected.
```

Comment:
this comment does not apply to `ViemWalletProvider` because these actions are only available for `CdpWalletProvider`